go-gitea/gitea
# Feature - #3031 - search for org repos

Adds the search form to the Org home page like it is for the User home page to search and sort repos of the Org. Addresses #3031 

Credit goes to @techknowlogick who made most all these edits in the closed PR #3081. Only updates I made were for the new sort options and new way to handle errors.

To test:
Check out branch, generate build, run gitea web, make a new org in your instance of Gitea and add multiple repos, staring and forking some of them. Then use the Search and Sort form on the Org home page.
